src/Microsoft.IdentityModel.Tokens/AsymmetricAdapter.cs

```diff
@@ -290,10 +290,30 @@ internal byte[] Encrypt(byte[] data)
 
 #if NET461 || NETSTANDARD1_4 || NETSTANDARD2_0
```

Comment:
> NET461 || NETSTANDARD1_4 || NETSTANDARD2_0 [](start = 3, length = 43)

nit/suggestion: should this have been a feature flag instead? Instead of having to write all of this again and again? - even more when adding a new target all the same places will have to be modified

Reply:
I like the feature flag approach.
Brent implied [here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1166#discussion_r284053291) that we will do some refactoring so we might leverage the feature flag approach there.